server/client-config/routes.js

```diff
@@ -12,4 +34,23 @@ module.exports = app => {
     res.type('js')
     res.send(response)
   })
+
+  app.get('/ping', nocache, async (req, res) => {
+    try {
+      const exists = await DBExists()
+      if (!exists) {
+        sendResult(res, 404, 'database error')
+        return
+      }
+      const { Body } = await s3.getObject({ Key: 'test-file' }).promise()
```

Comment:
@jyenes Should we call the `test-file` something more descriptive to avoid it getting deleted? For example `healthcheck-file` ?

Reply:
agree